docs/content/en/docs/implementing/container.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Working with container runtime
+description: How to run tasks in container runtime
+weight: 95
+---
+
+Container runtime allows the Keptn Lifecycle Toolkit
+to execute tasks in Kubernetes containers as part of a Kubernetes Job.
+This is similar to the Keptn v1 Job Executor Service.
+
+This is a new feature introduced in v0.8.0
+that is useful for:
+
+- Running load/performance testing tools
+- Running tasks expressed in other languages such as Python
+- [TODO] What else?
+- Running SLO validations
```

Comment:
It's not for SLO validations, for this we have KeptnEvaluations

Reply:
Yup, this is deleted